src/GovUk.Education.ExploreEducationStatistics.Admin/Services/Methodologies/MethodologyService.cs

```diff
@@ -400,6 +400,30 @@ await _context.Entry(loadedMethodology.ScheduledWithRelease)
                 });
         }
 
+        public async Task<Either<ActionResult, List<MethodologyVersionViewModel>>> ListMethodologyVersionsForApproval(Guid userId)
```

Comment:
I've added a comment to this effect here thanks.  I don't think that it's necessary in the MethodologyService and ReleaseService though, as we don't have any special case logic in there to mystify any potential readers of the code.  Do you think otherwise?